### PR TITLE
Added REST method for unignoring inbox entries.

### PR DESCRIPTION
**Pull request by dnobel**
Signed-off-by: Dennis Nobel <d.nobel@external.telekom.de>

### CODE REVIEW

File: bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/InboxResource.java [hunk 1]
```diff
@@ -108,6 +108,13 @@ public Response ignore(@PathParam("thingUID") String thingUID) {
         inbox.setFlag(new ThingUID(thingUID), DiscoveryResultFlag.IGNORED);
         return Response.ok().build();
     }
+    
+    @POST
+    @Path("/unignore/{thingUID}")
```
Comment (kaikreuzer) [inline bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/InboxResource.java]: shouldn't the endpoint better be inbox/<id>/unignore in a good REST manner (the ignore one of course as well then)?

Comment (dnobel) [inline bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/InboxResource.java]: Yes, I thought the same, but the endpoint for "ignore" also is "@Path("/ignore/{thingUID}")" (see the lines above). Maybe we should take the time to recheck the resource pathes. Is there any other application, which uses the new resources already? @cdjackson Does HABmin?

Comment (cdjackson) [inline bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/InboxResource.java]: Yes - HABmin is using the /ignore endpoint. To be honest - I don’t mind changing it if it’s deemed best…

Comment (dnobel) [inline bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/InboxResource.java]: Kai, in that case we can close this PR and I will create new one. I will check all endpoints to have the UID first and the action last.

**Comment by kaikreuzer** [comment]
closed as discussed.

**Pull request closed without merging**